README.md

```diff
@@ -75,6 +75,8 @@ Before extending into new frontiers, we need to improve the loaders API enough t
 - [ ] Hooks for customizing the REPL, including transpilation and tab completion. Support users pasting TypeScript (or CoffeeScript or whatever) into the REPL and having just as good an experience as with plain JavaScript.
 
    - [ ] Support top-level `await` in the REPL API, if possible.
+     
+   - [ ] Add support for `--eval` (or `--print`) CLI flags, extending over to support runtime `eval()` as well.
```

Comment:
Not `eval`; that doesn't support ESM syntax so it wouldn't make much sense to have hooks work there. We should support eval workers though.

Reply:
The benefit I see is adding TypeScript support

Here's an issue asking for support: https://github.com/privatenumber/tsx/issues/162

Reply:
Doesn't seem to support ESM either: https://github.com/nodejs/node/issues/30682

Reply:
But it could, like how we can have esm workers from files. We should link to that issue.

Reply:
I added it but later removed it entirely based on @aduh95 's request.

I hope you're okay with it, but if not, feel free to edit it back in.